developerportal/templates/robots.txt

```diff
@@ -2,3 +2,4 @@ User-agent: *
 Disallow: /admin/
 Disallow: /django-admin/
 Disallow: /static/
+Disallow: /search/
```

Comment:
I don't think you should do this. I don't see the reason. It's not harmful to allow spiders in there. It's not dangerous.

Reply:
Noted - I was being paranoid 😄 

- [ ] @stevejalim to remove this